### ISSUE TITLE
"Printed matter" in OpenSlides documents 

### ISSUE THREAD

**Issue by Lukas2112** [OpenSlides/OpenSlides#6137]
**Is your feature request related to a problem? Please describe.**
Missing but potentially very useful attribute for motions in OpenSlides.  

**Describe the solution you'd like**
Motions of the state or federal governments or of the opposition in the state or federal parliaments or the Bundesrat, for example, have an identifier, the so-called "Drucksache" (printed matter), by which a motion can be clearly identified. I could imagine that such an indicator for motions in OpenSlides could also be very useful to be able to find motions across sessions and legislatures.

**Describe alternatives you've considered**
Otherwise, you could manually add it to the PDF? 

**Additional context**
reference: https://www.mitmischen.de/bundestag-wissen/lexikon/d/drucksache#:~:text=Alle%20Dokumente%2C%20wie%20Gesetzentwürfe%2C%20Beschlussempfehlungen,%3A%20zum%20Beispiel%2019%2F11800.

**Comment by tsiegleauq** [comment]
While using OpenSlides <4, you can easily use the origin field for that. To rename "Herkunft" to "Drucksache" you can use custom translations. In OpenSlides >=4 the origin will be something else, but I agree that we should keep a free field for individual information.
[Motion1+Origin.pdf](https://github.com/OpenSlides/OpenSlides/files/6754536/Motion1%2BOrigin.pdf)

**Comment by Lukas2112** [comment]
Can the origin field be defined uniformly? So that "printed matter" would not have to be entered each time?

**Comment by tsiegleauq** [comment]
Has to be filled manually by editing the motion. In OpenSlides 4 it will be automatically filled after forwarding a motion to another meeting

**Comment by Lukas2112** [comment]
So in OS 4, a local parliament could, for example, define printed matter for documents across sessions, and embed their scheme, right?

**Comment by tsiegleauq** [comment]
Yes and no. The origin field cannot really defined by the user directly, it contains information about the first meeting where a motion was submitted, usually the motions original title, the original identifier and the meeting name where it was coming from

**Comment by Lukas2112** [comment]
Okay. That would come into play if motions were adjourned, right? A printed matter would also be quite complicated.

Unfortunately, I don't know and I can't accurately estimate how much effort it would take to have self-definable fields, such as "printed matter", "legislature" and then a field that makes the document unique. Could something like this perhaps be targeted for OS 4.1?
